src/components/CalculatorLine.js

```diff
@@ -26,52 +75,34 @@ function CalculatorLine() {
     return (
         <Row>
             <Col>
-                <FormControl
+                <InputField
                     placeholder="Nome do Fii"
-                    aria-label="name"
-                    onChange={createHandleChange(setName)}
+                    label="name"
+                    setFunction={setName}
+                    type='text'
                 />
             </Col>
             <Col>
-                <InputGroup>
-                    <InputGroup.Prepend>
-                        <InputGroup.Text>$</InputGroup.Text>
-                    </InputGroup.Prepend>
-                    <FormControl
-                        placeholder="Preço"
-                        aria-label="price"
-                        type='number'
-                        min='0'
-                        onChange={createHandleChange(setPrice)}
-                    />
-                </InputGroup>
+                <MoneyInputGroup
+                    placeholder="Preço"
+                    label="price"
+                    setFunction={setPrice}
+                />
             </Col>
             <Col>
-                <InputGroup>
-                    <InputGroup.Prepend>
-                            <InputGroup.Text>$</InputGroup.Text>
-                    </InputGroup.Prepend>
-                    <FormControl
-                        placeholder="Dividendos / Proventos"
-                        aria-label="price"
-                        type='number'
-                        min='0'
-                        onChange={createHandleChange(setDividends)}
-                    />
-                </InputGroup>
+                <MoneyInputGroup
+                    placeholder="Dividendos / Proventos"
+                    label="price"
+                    setFunction={setDividends}
+                />
             </Col>
             <Col>
-                <InputGroup>
-                    <InputGroup.Prepend>
-                        <InputGroup.Text>$</InputGroup.Text>
-                    </InputGroup.Prepend>
-                    <FormControl
-                        placeholder="Magic Number"
-                        aria-label="magic_number"
-                        readOnly={true}
-                        value={magicNumber}
-                    />
-                </InputGroup>
+                <MoneyInputGroup
+                    placeholder="Magic Number"
+                    label="magic_number"
+                    readOnly={true}
```

Comment:
```suggestion
                    readOnly
```
When the value is `true` you don't need to set the value, React does it for you